lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -256,7 +254,7 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
         ${_("Community TA's are members of the community whom you deem particularly "
         "helpful on the discussion boards. They can edit or delete any post, clear "
         "misuse flags, close and re-open threads, endorse responses, and see posts from "
-        "all cohorts. Their posts are marked as 'Community TA'. You can only give course "
+        "all groups. Their posts are marked as 'Community TA'. You can only give course "
```

Comment:
1) Remove the apostophe from "Community TA's" -> "Community TAs are..."

2) "You can only give course team roles to enrolled users." -> "Only enrolled users can be added as Discussion Admins."